vernac/library.ml

```diff
@@ -486,6 +499,11 @@ let save_library_struct ~output_native_objects dir =
     error_recursively_dependent_library dir;
   sd, md, vmlib, ast
 
+let save_library dir : library_t =
+  let sd, md, vmlib, _ast = save_library_struct ~output_native_objects:false dir in
+  let digest = Safe_typing.Dvo_or_vi (Digest.string "") in
```

Comment:
Is removing the dummy digest still todo?

Reply:
@ppedrot what should we do here?

Reply:
I'd put an option type and call it a day, or even just add a constructor to Safe_typing.vodigest.

Reply:
Ok, let me understand what purpose this thing serves, and I'll update it.

Reply:
This is used to check that the vo files didn't get modified under our feet. If you want added safety you could try to generate a unique number in the safe_typing API and check that instead of the hash of the data.

Reply:
I've opted to add a digest of the data, need to do some testing tho.